latest_majors.txt

```diff
@@ -0,0 +1,12 @@
+Latest DD major version of gofiber/fiber: v2
```

Comment:
Looks good to me!

Just for other readers if a _new_ major version is detected a line will get added like so

`New latest major version of ......`

which we can then use to prioritize work around